topsort-api-v2.yml

```diff
@@ -15,7 +19,7 @@ info:
 servers:
   - url: https://<slug>.api.sandbox.topsort.ai/v2
     description: Sandbox server (uses test data)
-  - url: https://<slug>.api.topsort.ai//v2
+  - url: https://<slug>.api.topsort.ai/v2
```

Comment:
can we set <slug> as a variable? with that, they can just fill it with the proper slug to test it :)

Reply:
I have no idea, currently `Try it!` is disabled for this API, I don't know why.